bin/rails

```diff
@@ -1,4 +1,17 @@
 #!/usr/bin/env ruby
-APP_PATH = File.expand_path('../../config/application', __FILE__)
-require_relative '../config/boot'
-require 'rails/commands'
+# frozen_string_literal: true
```

Comment:
Looks like someone is running ruby 2.3!  ❤️   But yes, let's not regenerate the binstubs in this PR.  😉